apps/heft/src/plugins/TypeScriptPlugin/TypeScriptPlugin.ts

```diff
@@ -19,7 +19,7 @@ import { TaskPackageResolver, ITaskPackageResolution } from '../../utilities/Tas
 import { JestTypeScriptDataFile } from '../JestPlugin/JestTypeScriptDataFile';
 import { ScopedLogger } from '../../pluginFramework/logging/ScopedLogger';
 import { ICleanStageContext, ICleanStageProperties } from '../../stages/CleanStage';
-import { HeftConfigFiles } from '../../utilities/HeftConfigFiles';
+import { ConfigFile } from '../../utilities/ConfigFile';
 
 const PLUGIN_NAME: string = 'typescript';
```

Comment:
Are plugins capitalized like `ClassNames` or like `folder-names`?

Reply:
(Maybe Heft should enforce something consistent.)

Reply:
We haven't enforced anything. I don't think it really matters that much, but we can add enforcement in another PR.